lua/nvim_lsp/util.lua

```diff
@@ -208,7 +208,7 @@ end)()
 
 -- Returns a function(root_dir), which, when called with a root_dir it hasn't
 -- seen before, will call make_config(root_dir) and start a new client.
-function M.server_per_root_dir_manager(make_config)
+function M.server_per_root_dir_manager(_make_config)
```

Comment:
```suggestion
function M.server_per_root_dir_manager(make_config)
```

Reply:
This avoids collisions with the function `make_config`. I prefer it the current way,